phoenixRest/views/card_order/__init__.py

```diff
@@ -60,8 +58,12 @@ def create_card_order(context, request):
     
     return card_order
 
-# Get all card orders for current event
+# Get all card orders for specified or current event
 @view_config(name="", context=CardOrderResource, request_method="GET", renderer="json", permission="view_all")
 def get_card_orders(context, request):
-    current_card_orders = request.db.query(CardOrder).filter(CardOrder.event == get_current_event(request)).all()
-    return current_card_orders
+    # We either get the specified event or the current event
+    if "event_uuid" in request.GET:
+        return request.db.query(CardOrder).filter(CardOrder.event_uuid == request.GET["event_uuid"]).all()
```

Comment:
1. I recommend being more consistent(why are you using two lines below doing what is 1 line here?)
2. If you only fetch the event object inside the if-statement and do the db fetch logic outside, you only need to maintain one sql query. We do this other places in the code base, i recommend it!